File: Selenium/Activities/Activity-2.py

from selenium import webdriver
from selenium.webdriver.common.by import By
with webdriver.Firefox () as driver:
    driver.get("https://training-support.net/webelements/login-form/")
    print("main page title: ",driver.title)
    driver.find_element(By.NAME,"username").send_keys("admin")
    driver.find_element(By.NAME,"password").send_keys("password")
    login = driver.find_element(By.XPATH,"//button[text()='Submit']")
    login.click()
   
    print("Login message: ",  driver.find_element(By.CSS_SELECTOR, "h1.text-center").text)
   # driver.quit()

File: Selenium/Activities/Activity_2.java
import org.openqa.selenium.By;
import org.openqa.selenium.WebDriver;
import org.openqa.selenium.firefox.FirefoxDriver;

public class Activity_2 {

	public static void main(String[] args) {
		
		WebDriver driver=new FirefoxDriver();
		driver.get("https://training-support.net/webelements/login-form");
		System.out.println("Main Page Title: "+driver.getTitle());
		driver.findElement(By.name("username")).sendKeys("admin");
		driver.findElement(By.name("password")).sendKeys("password");
		driver.findElement(By.xpath("//button[text()='Submit']")).click();
		System.out.println(driver.findElement(By.cssSelector("h1.text-center")).getText());
		driver.quit();
	}

}